shells/chrome/manifest.json

```diff
@@ -18,8 +20,7 @@
 
   "permissions": [
     "http://*/",
-    "https://*/*",
-    "file://*"
```

Comment:
Why did you remove `file://` permission? Seems like this would allow debugging on local files that don't go through http?